atom/browser/api/atom_api_screen.cc

```diff
@@ -107,7 +122,8 @@ mate::ObjectTemplateBuilder Screen::GetObjectTemplateBuilder(
       .SetMethod("getPrimaryDisplay", &Screen::GetPrimaryDisplay)
       .SetMethod("getAllDisplays", &Screen::GetAllDisplays)
       .SetMethod("getDisplayNearestPoint", &Screen::GetDisplayNearestPoint)
-      .SetMethod("getDisplayMatching", &Screen::GetDisplayMatching);
+      .SetMethod("getDisplayMatching", &Screen::GetDisplayMatching)
+      .SetMethod("getDesktopCapturer", &Screen::GetDesktopCapturer);
```

Comment:
The `screen` module is also available on renderer process, so having the desktop capturer in the `screen` module is probably a bad idea, I think it is fine to make it an independent module.